### PR TITLE
Add HGLR-ZEUSF722 AIO.config

### PR DESCRIPTION
**Pull request by HGLRC-D**
Hi,

please add this target for HGLRC.

### CODE REVIEW

**Comment by HGLRC-D** [comment]
Can someone tell me why the check failed?

**Comment by HGLRC-D** [comment]
ZEUSF722 AIO is an integrated ESC based on F722.config. The layout of the gyroscope is different from the past. In order to facilitate the user to use and distinguish, request to add ZEUSF722 AIO.config.

**Comment by etracer65** [comment]
The file name can't have spaces.

**Comment by HGLRC-D** [comment]
> The file name can't have spaces.

Thank you very much！

File: configs/default/HGLR-ZEUSF722_AIO.config [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+# serial
+serial 0 64 115200 57600 0 115200
+
+# master
+set mag_bustype = I2C
+set mag_i2c_device = 1
+set baro_spi_device = 1
+set blackbox_device = SPIFLASH
+set dshot_burst = ON
+set motor_pwm_protocol = DSHOT600
```
Comment (mikeller) [inline configs/default/HGLR-ZEUSF722_AIO.config:119]: I assume that this board will only ever be sold as an all-in-one with the ESCs on board?

Comment (HGLRC-D) [reply configs/default/HGLR-ZEUSF722_AIO.config:119]: Yes, ZEUSF722 AIO is based on F722 FC integrated BLHELI32 ESC.

**Review by mikeller — APPROVED** [review]

**Pull request merged**